.gitignore

```diff
@@ -1,7 +1,8 @@
 /node_modules
```

Comment:
This comment has nothing to do with this file, but didn't know where else to put it: I see that you renamed a few files as part of this PR that don't really have anything to do with this PR- next time include in the PR things that are _exclusively_ part of the ticket description!